src/extensions/averagewaveforms/AverageWaveformsView/fetch_average_waveforms_2.py

```diff
@@ -23,36 +23,20 @@ def createjob_fetch_average_waveform_2(labbox, recording_object, sorting_object,
             unit_id=unit_id
         )
 
-@hi.function('fetch_average_waveform_2', '0.2.10')
+@hi.function('fetch_average_waveform_2', '0.2.14')
 @hi.container('docker://magland/labbox-ephys-processing:0.3.19')
 @hi.local_modules([os.getenv('LABBOX_EPHYS_PYTHON_MODULE_DIR')])
 @le.serialize
 def fetch_average_waveform_2(snippets_h5, unit_id):
     import h5py
     h5_path = ka.load_file(snippets_h5)
-    with h5py.File(h5_path, 'r') as f:
-        unit_ids = np.array(f.get('unit_ids'))
-        channel_ids = np.array(f.get('channel_ids'))
-        channel_locations = np.array(f.get(f'channel_locations'))
-        sampling_frequency = np.array(f.get('sampling_frequency'))[0].item()
-        if np.isnan(sampling_frequency):
-            print('WARNING: sampling frequency is nan. Using 30000 for now. Please correct the snippets file.')
-            sampling_frequency = 30000
-        unit_spike_train = np.array(f.get(f'unit_spike_trains/{unit_id}'))
-        unit_waveforms = np.array(f.get(f'unit_waveforms/{unit_id}/waveforms'))
-        unit_waveforms_channel_ids = np.array(f.get(f'unit_waveforms/{unit_id}/channel_ids'))
-
-        print(unit_waveforms_channel_ids)
+    unit_waveforms, unit_waveforms_channel_ids, channel_locations0, sampling_frequency, unit_spike_train = le.get_unit_waveforms_from_snippets_h5(h5_path, unit_id)
```

Comment:
uses the new get_unit_waveforms_from_snippets_h5 which supports unit_id being a list.